setup.py

```diff
@@ -490,7 +490,8 @@ def pxd(name):
     index={'pyxfile': 'index',
            'sources': ['pandas/src/datetime/np_datetime.c',
                        'pandas/src/datetime/np_datetime_strings.c'],
-           'pxdfiles': ['src/util']},
+           'pxdfiles': ['src/util'],
+           'depends': _pxi_dep['index'] + _pxi_dep['algos']},
```

Comment:
well this is dependent on ``pandas.algos``, but a change in ``algos.pyx`` doesn't actually require re-compilation of ``index.pyx`` I don't think.